driver/ppm_flag_helpers.h

```diff
@@ -2201,4 +2201,34 @@ static __always_inline uint32_t mknod_mode_to_scap(uint32_t modes)
 	return res;
 }
 
-#endif /* PPM_FLAG_HELPERS_H_ */
+static __always_inline uint32_t bpf_cmd_to_scap (unsigned long cmd){
```

Comment:
This helper was already written as `cmd` was already an enum value; i'd just add the new switch cases.

Reply:
I think we can also do what eg: we did for `fsconfig_cmds_to_scap`, since `bpf` cmd are defined as uapi, they won't ever change (NEVER BREAK USER API rule by Torvalds itself :D), therefore a `return cmd` is enough here (if and only if our `PPM_BPF_` values do perfectly match kernel `BPF_` values!)

Reply:
Yes, this makes sense!!

Reply:
I liked @FedeDP suggestion, but now I wonder why we even touch the driver code at all, and why we don't just do userspace conversions? Adding this statement to the userspace parsing would be nice!